packages/chat-sdk/src/channel/channel.ts

```diff
@@ -983,27 +985,122 @@ export class Channel implements BaseChannel {
     );
   }
 
+  /**
+   * Remove chat cofig changes listener
+   *
+   */
+  private offChatConfigListener(callback?: (item: any) => void, type?: ChatConfigType): void {
```

Comment:
Those parameters should be required. The `item` type should be `LiveChatChannel | ChatMessage`.